components/work_templates/index.tsx

```diff
@@ -134,13 +145,35 @@ const WorkTemplateModal = () => {
         setSelectedVisibility(visibility);
     };
 
-    const create = (template: WorkTemplate, name = '') => {
-        const str = `TODO: creating elements from template ${template!.id} with name ${name || '<empty>'}`;
+    const create = async (template: WorkTemplate, name = '', visibility = Visibility.Public) => {
+        const pbTemplates = template.content.filter((item) => item.playbook).map((item) => {
+            return playbookTemplates.find((pb) => pb.title === item.playbook!.template)!;
+        });
```

Comment:
nit: A rare use case but this could end up returning `[undefined, ...]`.
Maybe a `for...of` could bring more control over the returned array, also reducing nested loops.

Reply:
updated!